catalogue/terraform/stack/main.tf

```diff
@@ -42,6 +46,17 @@ module "path_listener" {
   priority      = "49997"
 }
 
+module "api_path_listener" {
+  source = "../../../infrastructure/modules/alb_listener_rule"
+
+  alb_listener_https_arn = var.alb_listener_https_arn
+  alb_listener_http_arn  = var.alb_listener_http_arn
+  target_group_arn       = local.target_group_arn
+
+  path_patterns = ["/api/works*"]
```

Comment:
N.B. tried using next.js rewrites, so this wouldn't be necessary but it doesn't work with the api routes